### PR TITLE
Update OCF Cloud API for Cloud Services documentation

### PR DESCRIPTION
**Pull request by Danielius1922**
Add Origin Cloud test cases to the documentation.

### CODE REVIEW

File: content/en/tutorials/ocf-conformance-testing.md [hunk 1]
```diff
@@ -174,3 +178,461 @@ The steps needed to compile and run `cloud_tests` device with plgd hub #bundle's
...
+
+Additional, several of the Origin Cloud test cases require different [setup](https://plgd.dev/configuration/cloud2cloud-connector) of the `cloud2cloud-connector` and also different values in the PICS configuration. Only with the correct configuration of each, the test cases will succeed.
+
+### Executing Origin Cloud API commands using Postman
+
+After starting the bundle as described in [Run plgd hub #bundle in Ubuntu machine](#run-plgd-hub-bundle-in-ubuntu-machine) the `cloud2cloud-connector` service is available at the address ```https://{FQDN}/c2c/connector```.
+
+#### Prepare environment
+
+In the sidebar select **Environment**, open the **Globals** pane and setup three variables there:
```
Comment (ondrejtomcik) [inline content/en/tutorials/ocf-conformance-testing.md:193]: typo - pane

File: content/en/tutorials/ocf-conformance-testing.md [hunk 2]
```diff
@@ -174,3 +178,461 @@ The steps needed to compile and run `cloud_tests` device with plgd hub #bundle's
...
+| CLIENT_SECRET | `s`                                   |
+| NGROK_URL     | `https://4561-95-102-120-86.ngrok.io` |
+
+For `FQDN` use the value obtained in [Run plgd hub #bundle in Ubuntu machine](#run-plgd-hub-bundle-in-ubuntu-machine). `CLIENT_SECRET` can be any non-empty value. `NGROK_URL` value is provided by the ngrok application as described in the _User Guide_ section **Setup for all Origin Cloud test cases**.
+
+![Global variables in Postman](/images/tutorials/ocf-conformance-testing-postman-globals.png)
+
+#### Get Authorization Token
+
+Each command requires an authorization header with valid JWT token, otherwise it will fail. We can use Postman to request and store a JWT token, which we can then append to various commands. To request a token create a new HTTP Request (`CTRL+N` and then select **HTTP Request**). In the horizontal menu of the new request select the **Authorization** pane. Set the type of the token to **OAuth 2.0** and for the new token configure the following options:
```
Comment (ondrejtomcik) [inline content/en/tutorials/ocf-conformance-testing.md:208]: typo - pane

**Review by ondrejtomcik — APPROVED** [review]
Crazy steps :D CTT should improve a bit ... Good job @Danielius1922

File: content/en/tutorials/ocf-conformance-testing.md (hunk 1 above)
Comment (Danielius1922) [reply content/en/tutorials/ocf-conformance-testing.md:193]: What word are you thinking of? Panel? 

Pane is a word (https://www.merriam-webster.com/dictionary/pane) though, I've used it as synonym for section.

Comment (ondrejtomcik) [reply content/en/tutorials/ocf-conformance-testing.md:193]: LOL, okay :) My fail.

**Pull request merged**